docs/preview/features/correlation.md

```diff
@@ -22,7 +22,7 @@ This correlation is based on the `RequestId` and `X-Transaction-ID` HTTP request
 
 ## How This Works
 
-When an application is configured to use the default configuration of the correlation, each HTTP response will get an extra header called `RequestId` containing an unique identifier to distinguish between requests/responses.
+When an application is configured to use the default configuration of the correlation, each HTTP response will get an extra header called `RequestId` containing an unique identifier to distinguish between requests/responses. This ID will act as the parent ID for all telemetry that comes after and uses the [HTTP Correlation](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Diagnostics.DiagnosticSource/src/HttpCorrelationProtocol.md) to extract the most recent ID.
```

Comment:
```suggestion
When an application is configured to use the default configuration of the correlation, each HTTP **response** will have an extra header called `RequestId` containing an unique identifier to distinguish between requests/responses. This ID will act as the parent ID for all telemetry that comes after and uses the [HTTP Correlation](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Diagnostics.DiagnosticSource/src/HttpCorrelationProtocol.md) to extract the most recent ID.
```
Made it more clear that it is the response that has an extra header.
The sentencee that was added, is not quite clear imho though.   I think you mean that all requests to other dependencies inside the request, use this id as parent id ?

Reply:
Yes, although I had problems reading your comment 😄 .